### PR TITLE
chore: Update aws-swift-sdk to 0.5.0

### PR DESCRIPTION
**Pull request by harsh62**
*Issue #, if available:*

*Description of changes:*

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [x] Build succeeds with all target using Swift Package Manager
- [x] All unit tests pass
- [x] All integration tests pass
- [x] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [x] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by phantumcode — DISMISSED** [review]

File: AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Support/Extensions/Date+Formatting.swift [hunk 1]
```diff
@@ -15,6 +15,14 @@ extension DateFormatter {
         formatter.locale = Locale(identifier: "en_US_POSIX")
         return formatter
     }()
+
+    static let iso8601DateFormatterWithFractionalSeconds: DateFormatter = {
```
Comment (royjit) [inline AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Support/Extensions/Date+Formatting.swift:19]: where is this used?

**Review by royjit — DISMISSED** [review]
LGTM, one minor comment

File: AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Support/Extensions/Date+Formatting.swift (hunk 1 above)
Comment (harsh62) [reply AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Support/Extensions/Date+Formatting.swift:19]: This is used in Analytics tests.

**Comment by harsh62** [comment]
Integration tests are passing: https://github.com/aws-amplify/amplify-swift/actions/runs/3484899242

**Review by phantumcode — APPROVED** [review]

**Pull request merged**